python/lsst/ap/pipe/selectImages.py

```diff
@@ -59,27 +59,27 @@ def runDataRef(self, dataRef, coordList, makeDataRefList=True, selectDataList=[]
 
         Returns
         -------
-        pipe.base.Struct with filtered exposureList and dataRefList
-        (if makeDataRefList is True).
+        result : `lsst.pipe.base.Struct`
+            Result struct with components:
+
+            - ``exposureList``: the selected exposures
+                (`list` of `lsst.pipe.tasks.selectImages.BaseExposureInfo`).
+            - ``dataRefList``: the optional data references corresponding to
+                each element of ``exposureList``
+                (`list` of `lsst.daf.persistence.ButlerDataRef`, or `None`).
 
         Notes
         -----
-        We use the "convexHull" function in the geom package to define
+        We use the "convexHull" method in the `~lsst.sphgeom` package to define
         polygons on the celestial sphere, and test the polygon of the
         patch for overlap with the polygon of the image.
-
-        We use "convexHull" instead of generating a SphericalConvexPolygon
-        directly because the standard for the inputs to SphericalConvexPolygon
-        are pretty high and we don't want to be responsible for reaching them.
-        If "convexHull" is found to be too slow, we can revise this.
```

Comment:
I don't entirely understand the context for this original comment, but given that the entirety of `selectImages.py` will be removed as part of DM-11953, I'm not overly concerned about it.

Reply:
`lsst.geom` had a "with checks" and a "without checks" way to get a convex polygon. `lsst.sphgeom` always checks, as far as I can tell.